demos/OH-EXF-Kamakura/README.md

```diff
@@ -1,5 +1,5 @@
-# EdgeX Foundry Jakarta w/ Onvif Device Service + Edge Video Analytics
-Deployment files for running EdgeX Jakarta with the onvif device service, as well
+# EdgeX Foundry Kamakura w/ Onvif Device Service + Edge Video Analytics
```

Comment:
What about `Camera management app`, it also needs to be specified here right?

Reply:
yes